src/pages/gen2/build-a-backend/data/customize-authz/public-data-access/index.mdx

```diff
@@ -57,7 +57,7 @@ const schema = a.schema({
     .model({
       content: a.string(),
     })
-    .authorization([a.allow.public('identityPool')]),
+    .authorization(allow => [allow.guest()]),
```

Comment:
For line 52 above, do we need to update the part about `identityPool`?

Reply:
Nope not needed. guest == public('identityPoo'). publicApiKey === public()